docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudwatch-firehose.asciidoc

```diff
@@ -0,0 +1,270 @@
+4. As a **Runtime**, select a recent version of Python (for example, Python 3.11).
+5. Select your **Architecture** of choice between `arm64` and `x86_64`.
+6. Confirm and create the Lambda function
+
+When AWS completes the creation of the function, visit the **Code source** section and paste the following Python code as function source code:
+
+[source,python]
+----
+import json
+
+
+def lambda_handler(event, context):
+    print("Received event: " + json.dumps(event))
+----
+
+[IMPORTANT]
+=====
+Click on **Deploy** to deploy the changes to the source code.
+=====
+
+[discrete]
+[[firehose-cloudwatch-step-two-genereate-sample-logs]]
+=== Generate some sample logs
+
+With the function ready to go, we can invoke it a few times to generate sample logs.
+
+On the function page,
+
+- Select **Test**
+- Select the option to create a new test event
+- Name the test event and **Save** the changes.
+- Click on the **Test** button to execute the function.
+
+Visit the function's log group. Usually, the AWS console offers a handy link to jump straight to the log group it created for this function's logs.
+
+You should see something similar:
+
+image::firehose-cloudwatch-sample-logs.png[CloudWatch log group with sample logs]
+
+Take note of the log group name for this Lambda function, and move to the next section.
+
+[discrete]
+[[firehose-cloudwatch-step-three]]
+== Step 3: Create a stream in Amazon Data Firehose
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later. 
+
+NOTE: For advanced use cases, source records can be transformed by invoking a custom Lambda function. When using Elastic integrations, this should not be required.
+
+Set the **Parameters** in the **Destination settings** section.
+
+[discrete]
+[[firehose-cloudwatch-step-three-desination-settings-parameters]]
+=== Parameters
+
+Use the following parameters:
+
+
+[cols="1,1",options="header"]
+|===
+|Name
+|Value
+
+| `es_datastream_name`
+| `logs-aws.generic-default`
+
+|===
+
+[discrete]
+[[firehose-cloudwatch-step-four]]
+== Step 4: Set up a subscription filter to send Lambda function log events to a Firehose stream
+
+The Firehose stream is ready to send logs to our Elastic Cloud deployment.
+
+Next steps:
+
+- Open the log group with the Lambda function log events
+- Create a subscription filter for Amazon Data Firehose 
+
+[discrete]
+[[firehose-cloudwatch-step-four-log-group]]
+=== Open the log group with the Lambda function log events
+
+Please open the log group where the Lambda service is sending the events. We must forward these events to an Elastic stack using the Firehose delivery stream.
```

Comment:
I'm suggesting "can" instead of "must" here because it feels like we're saying that firehose is the only way to get the logs into Elastic.
```suggestion
Open the log group where the Lambda service is sending the events. You can forward these events to an Elastic stack through the Firehose delivery stream.
```